aptos-move/framework/aptos-framework/sources/staking_contract.spec.move

```diff
@@ -5,32 +5,42 @@ spec aptos_framework::staking_contract {
     }
 
     spec stake_pool_address(staker: address, operator: address): address {
-        include StakingContractExistsAbortsIf;
+        include ContractExistsAbortsIf;
     }
 
     /// Staking_contract exists the stacker/operator pair.
     spec last_recorded_principal(staker: address, operator: address): u64 {
-        include StakingContractExistsAbortsIf;
+        include ContractExistsAbortsIf;
     }
 
     /// Staking_contract exists the stacker/operator pair.
     spec commission_percentage(staker: address, operator: address): u64 {
-        include StakingContractExistsAbortsIf;
+        include ContractExistsAbortsIf;
     }
 
     /// Staking_contract exists the stacker/operator pair.
     spec staking_contract_amounts(staker: address, operator: address): (u64, u64, u64) {
-        // Not verified when verify_duration_estimate > vc_timeout
-        pragma verify_duration_estimate = 120; // TODO: set because of timeout (property proved).
+        // TODO: verfication timeout
```

Comment:
If the property is proved, please use the comment "// TODO: set because of timeout (property proved)."